Functions/Functions.cs

```diff
@@ -0,0 +1,79 @@
+
+using Godot;
+using System;
+using System.Collections.Generic;
+
+namespace Functions {
+
+    /// <summary>
+    /// For Debugging purposes. If <c> Enabled = true </c>, debug output related <br/>
+    /// to timing and playbak status will be enabled.
+    /// </summary>
+    public static class Debugging {
+        public static readonly bool Enabled = false;
+    }
+
+    /// <summary>
+    /// Class for Sycnhronous Timing within the application. 
+    /// </summary>
+    public partial class TimeKeeper {};
```

Comment:
Do you need this line here? Seems unnecessary

Reply:
I put that there in the same vane that I included `Function `and all the others in the `Functions `namespace. I was under the impression that each file I put the `namespace Functions` declaration in, I should also include in the actual namespace file(?).

Reply:
There is no namespace file. Namespaces are folders, usually. You can just put `namespace X.Y.Z;` at the top of a file and it's fine